Pyscript blog v2

connorcasner opened the pull request.

toonarmycaptain commented:
Thankyou! This looks much better with the full width. 
The one note I would make is that the colour of `print(` is not so easy (for me to read), and the error message is very hard to make out unless you highlight the text with the cursor:
![Screenshot from 2022-07-08 23-13-39](https://user-images.githubusercontent.com/29956894/178091220-bf766939-5884-4b13-aa1c-71166498ab2c.png)
Might just be my eyes, but I don't know how easy it is to change the palette of the syntax highlighting/autocomplete?

nixjdm commented:
@connorcasner Did the first commit adding the pyscript css contain any changes to the css, or is https://github.com/terminal-labs/tl_web/pull/40/commits/c33363dd3e55b672384da3c3dc7e36a49d56f427 the only changes you made? Can you tell us exactly where you got the starting pyscript css file?

I want to change the name of the file as it's stored in our repo to `tl-pyscript-{short commit hash of original}-{date}.css`. I also want to vendor the pyscript js file and name it the same way. The reason is that I want to make sure

- that this blog post works indefinitely
- other pyscript uses on our site are possible in the future with or without this version of the css or js.
- that we can fairly easily discern what our changes are

connorcasner commented:
Added the commit sha of the original pyscript CSS to the tl-pyscript-db39207-2022-07-06.css file and added a disclaimer explaining  why we vendor the pyscript css.
